azext_edge/edge/_help.py

```diff
@@ -716,15 +716,15 @@ def load_iotops_help():
     """
 
     helps[
-        "iot ops secretsync show"
+        "iot ops secretsync list"
     ] = """
         type: command
-        short-summary: Show the secret sync config associated with an instance.
+        short-summary: List the secret sync configs associated with an instance.
 
         examples:
-        - name: Show the secret sync config associated with an instance.
+        - name: List the secret sync configs associated with an instance.
           text: >
-            az iot ops secretsync show --name myinstance -g myresourcegroup
+            az iot ops secretsync list --name myinstance -g myresourcegroup
     """
 
     helps[
```

Comment:
In the Disable summary can we append something like all SPCs and related secret sync resources will be deleted ?

Reply:
yeah sounds good, I will update

Reply:
<img width="442" alt="image" src="https://github.com/user-attachments/assets/b92ad2d7-9ed6-4934-9f2a-33fdd5c174b6">

Reply:
<img width="437" alt="image" src="https://github.com/user-attachments/assets/789ca61d-4b9c-41e6-bea1-d534988635ac">